### DISCUSSION TITLE
I can't access getPlaneDiscoveryController

### DISCUSSION THREAD

**Discussion by MihajloAndrejicNis** [sceneview/sceneform-android#154] [Q&A]
I tried to change image of hand when plane is detecting accessing  getPlaneDiscoveryController.
Code is like this:
arFragment.getPlaneDiscoveryController().setInstructionView(phoneImage);

**Comment by ThomasGorisse** [comment]
`PlaneDiscoveryController` has been replaced by `InstructionsController` which can also handle the AugmentedImage detection instructions.
- If you only want to change the hand image, simply override the `sceneform_hand_phone.png` in your drawable folder.
- If you want a completely specific Instructions view, override the `InstructionsController.onCreateView(int type)` and apply it to your fragment:
```java
arFragment.setInstructionsController(new InstructionsController(inflater, frameLayout) {
    @Override
    protected View onCreateView(int type) {
        switch (type) {
            case TYPE_PLANE_DISCOVERY:
                // return your view here
            default:
                return super.onCreateView(type);
        }
    }
});
```

**Comment by MihajloAndrejicNis** [comment]
Thank you very much, the first solution is what I needed.
